tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -988,16 +988,14 @@ IncludeByKey.PASSTHROUGH {
 					direction	"playback"
 					name "SSP2 Playback"
 					formats 'S16_LE,S24_LE,S32_LE'
-					rate_min 8000
-					rate_max 192000
+					rates "8000,11025,16000,22050,32000,44100,48000,64000,88200,96000,176400,192000"
 				}
 
 				Object.PCM.pcm_caps.2 {
 					direction	"capture"
 					name "SSP2 Capture"
 					formats 'S16_LE,S24_LE,S32_LE'
-					rate_min 8000
-					rate_max 192000
+					rates "8000,11025,16000,22050,32000,44100,48000,64000,88200,96000,176400,192000"
```

Comment:
do we support 44.1? Not sure how the clocks would be derived or maybe this assumes an SRC?

Reply:
Yes,  there's a SRC in host pipeline for SSP2. I'm trying to replicate with this topology an issue with 44.1 kHz volume with HiFi4 and noticed other than 48 kHz no more worked.